docs/se/config/advanced-configuration.adoc

```diff
@@ -259,37 +255,28 @@ sources gives your application a way to access all config elements distinctly ev
 if their keys would otherwise conflict.
 
 ===== Merging Strategies
-The `ConfigSources.create(Supplier<ConfigSource>...)` and `ConfigSources.create(List<Supplier<ConfigSource>...)`
-methods return a `CompositeBuilder`.
-By default, earlier sources in the list have higher priority than later ones, meaning
+When creating config from multiple sources it is possible, that the same key comes from multiple
```

Comment:
When creating config from multiple sources, it is possible that the same key comes from multiple sources.